Add missing german tanslations

elkuku opened the pull request:
Some missing translations.

Do you have any kind of translation policies and/or online translation tools currently enabled for this project? (e.g. Transifex, Crowdin, etc.)

javiereguiluz commented:
Thanks for contributing these translations. I'll wait for the review of our German-speaking friends. About your question: we don't have any policy about this or use any of the tools you mentioned. Thanks!

xabbuh commented on app/Resources/translations/messages.de.xlf:
```diff
@@ -276,6 +276,30 @@
+            </trans-unit>
+            <trans-unit id="rss.title">
+                <source>rss.title</source>
+                <target>Symfony Demo blog</target>
```
Blog

xabbuh commented on app/Resources/translations/messages.de.xlf:
```diff
@@ -276,6 +276,30 @@
+            </trans-unit>
+            <trans-unit id="rss.description">
+                <source>rss.description</source>
+                <target>Die neusten Posts die auf dem Symfony Demo Blog veröffentlicht wurden</target>
```
Die neuesten Posts, die auf dem Symfony Blog veröffentlicht wurden.

elkuku commented:
Fixed :wink:

elkuku commented:
The security check is failing on my language file changes - I love Travis :tongue:

elkuku commented:
While playing wit #531 I discovered more missing strings... @xabbuh would you mind to re-check? :wink:

xabbuh commented on app/Resources/translations/messages.de.xlf:
```diff
@@ -276,6 +276,78 @@
+            </trans-unit>
+            <trans-unit id="http_error.description">
+                <source>http_error.description</source>
+                <target>Ein unbekannter Fehler (HTTP %status_code%) ist aufgetreten wegen dem ihre Anfrage nicht ausgeführt werden konnte.</target>
```
I would reword this:

> Ein unbekannter Fehler (HTTP %status_code%) ist aufgetreten. Ihre Anfrage konnte deswegen nicht ausgeführt werden.

xabbuh replied:
By the way, it looks like everywhere else we used "du" instead of "Sie".

xabbuh commented on app/Resources/translations/messages.de.xlf:
```diff
@@ -276,6 +276,78 @@
+            </trans-unit>
+            <trans-unit id="http_error_404.suggestion">
+                <source>http_error_404.suggestion</source>
+                <target>Überprüfen Sie Rechtschreibfehler in der URL or <a href="%url%">gehen sie zurück zur Homepage</a>.</target>
```
or -> oder

xabbuh commented on app/Resources/translations/messages.de.xlf:
```diff
@@ -276,6 +276,78 @@
+            </trans-unit>
+            <trans-unit id="http_error_500.suggestion">
+                <source>http_error_500.suggestion</source>
+                <target>Versuchen sie diese Seite in einigen Minuten erneut zu laden or <a href="%url%">gehen sie zurück zur Homepage</a>.</target>
```
or -> oder

xabbuh commented on app/Resources/translations/messages.de.xlf:
```diff
@@ -276,6 +276,78 @@
+            </trans-unit>
+            <trans-unit id="notification.comment_created.description">
+                <source>notification.comment_created.description</source>
+                <target>Ihr Beitrag "%title%" hat einen Kommentar erhalten. Sie können den Kommentar lesen venn sie <a href="%link%">diesem Link</a> folgen.</target>
```
[...] lesen, wenn [...]

elkuku commented:
Fixed, Thanks!

I also changed everything to "informal".

BTW: @xabbuh do you think #531 makes any sense? How do *you* check for missing translations? Don't you think using an online translation service (Transifex, Crowdin, etc.) would help?

Thanks again :wink:

javiereguiluz commented:
Thank you @elkuku.
